### PR TITLE
Add the first set of PIDs from E1.37-1

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: firmware/src/dimmer_model.c [hunk 1]
```diff
@@ -40,19 +42,72 @@ static const char DEFAULT_DEVICE_LABEL[] = "Ja Rule";
 static const char PERSONALITY_DESCRIPTION[] = "Dimmer";
 static const uint16_t INITIAL_START_ADDRESSS = 1u;
 
+static const char LOCK_STATE_DESCRIPTION_UNLOCKED[] = "Unlocked";
+static const char LOCK_STATE_DESCRIPTION_SUBDEVICE_LOCKED[] =
```
Comment (peternewman) [inline firmware/src/dimmer_model.c]: SPAG SUBDEVICES

File: firmware/src/dimmer_model.c [hunk 2]
```diff
@@ -161,24 +251,327 @@ int DimmerModel_SetDMXBlockAddress(const RDMHeader *header,
...
+    return RDMResponder_BuildNack(header, NR_DATA_OUT_OF_RANGE);
+  }
+
+  g_root_device.pin_code = new_pin;
+  return RDMResponder_BuildSetAck(header);
+}
+
+int DimmerModel_GetLockState(const RDMHeader *header,
+                             UNUSED const uint8_t *param_data) {
+  uint8_t *ptr = g_rdm_buffer + sizeof(RDMHeader);
```
Comment (peternewman) [inline firmware/src/dimmer_model.c]: This must fail GetLockStateWithData?

Comment (peternewman) [inline firmware/src/dimmer_model.c]: Ah it's done elsewhere now isn't it.

**Comment by peternewman** [comment]
Don't you need a task to decrement the burn in?

File: firmware/src/dimmer_model.c (hunk 1 above)
Comment (nomis52) [inline firmware/src/dimmer_model.c]: Done

File: firmware/src/dimmer_model.c (hunk 2 above)
Comment (nomis52) [inline firmware/src/dimmer_model.c]: Yup

**Comment by nomis52** [comment]
> Don't you need a task to decrement the burn in?

Ideally we would but it's more complexity than I want to support right now.

**Comment by peternewman** [comment]
Can you add a todo for the burn in at least, then LGTM

**Comment by nomis52** [comment]
Done

**Comment by peternewman** [comment]
LGTM

**Pull request merged**